src/main/target/common_fc_pre.h

```diff
@@ -82,7 +82,9 @@
 #define FAST_CODE
 #endif
 
-#ifdef USE_FAST_RAM
+#define USE_FAST_DATA_RAM
+
+#ifdef USE_FAST_DATA_RAM
```

Comment:
Does this (and all TCP) directives belong here?

Reply:
@mikeller   Did you put stm32_flash_f722_no_split.ld into target folder and modify target.mk

Reply:
No I did not. I think this should work without a custom target config for it.